src/extract_metadata.rs

```diff
@@ -38,7 +34,7 @@ mod tests {
             .join("IMG_20190804_152120.jpg");
 
         // when
-        let metadata = PhotoMetadata::from_file(&path);
+        let metadata = PhotoMetadata::from_file(&path).unwrap();
 
         // then
         assert_eq!(metadata.datetime.year, 2019);
```

Comment:
no ten element prosi się o custom asercje w sofcie, który głównie opiera się na tych datach ;)

Reply:
nie rozumiem

Reply:
chodzi o to, że sprawdzasz każdy element daty po kolei, zamiast zrobić jakiś prosty helper assert_datetime_eq!(metadata.datetime, 2019, 12, 10, 13, 30), albo jakoś ładniej ;)